tm2/pkg/crypto/ledger/ledger_secp256k1.go

```diff
@@ -177,6 +178,10 @@ func convertDERtoBER(signatureDER []byte) ([]byte, error) {
 }
 
 func getLedgerDevice() (ledger.SECP256K1, error) {
+	if !hid.Supported() {
+		return nil, fmt.Errorf("Ledger support is not enabled. Try building with CGO_ENABLED=1")
```

Comment:
```suggestion
		return nil, fmt.Errorf("ledger support is not enabled, try building with CGO_ENABLED=1")
```

(Error strings should not be capitalized)